SIDBlasterLib/SIDBlasterInterfaceImpl.cpp

```diff
@@ -173,13 +173,14 @@ SIDBlasterEnumerator::SIDBlasterEnumerator() {
 }
 
 SIDBlasterEnumerator::~SIDBlasterEnumerator() {
-  delete m_Manager;
+	D2XXManager::destroy();
 }
 
 int   
 SIDBlasterEnumerator::DeviceCount() {
   if (m_Manager == NULL) {
-    m_Manager = new D2XXManager();
+    //m_Manager = new D2XXManager();
+	D2XXManager* m_Manager = D2XXManager::getInstance();
```

Comment:
If m_Manager should no longer be a member of SIDBlasterEnumerator, it should be removed from the class declaration. Also rename the local variable (m_Manager -> manager) to avoid confusion.